samples/stock/demo/relative-rotation-graph/demo.js

```diff
@@ -0,0 +1,289 @@
+                color: '#0000ff'
+            },
+            point: {
+                x: 96,
+                y: 104,
+                xAxis: 0,
+                yAxis: 0
+            }
+        }]
+    }, {
+        draggable: false,
+        zIndex: 0,
+        shapes: [{
+            fill: 'rgba(0, 255, 0, 0.1)',
+            type: 'path',
+            points: [{
+                x: 100,
+                y: 104,
+                xAxis: 0,
+                yAxis: 0
+            }, {
+                x: 104,
+                y: 104,
+                xAxis: 0,
+                yAxis: 0
+            }, {
+                x: 104,
+                y: 100,
+                xAxis: 0,
+                yAxis: 0
+            }, {
+                x: 100,
+                y: 100,
+                xAxis: 0,
+                yAxis: 0
+            }]
+        }],
+        labels: [{
+            text: 'Leading',
+            backgroundColor: 'transparent',
+            borderWidth: 0,
+            y: 0,
+            style: {
+                fontSize: '16px',
+                fontWeight: 'bold',
+                color: '#00ff00'
+            },
+            point: {
+                x: 104,
+                y: 104,
+                xAxis: 0,
+                yAxis: 0
+            }
+        }]
+    }, {
+        draggable: false,
+        zIndex: 0,
+        shapes: [{
+            fill: 'rgba(255, 255, 0, 0.1)',
+            type: 'path',
+            points: [{
+                x: 100,
+                y: 100,
+                xAxis: 0,
+                yAxis: 0
+            }, {
+                x: 104,
+                y: 100,
+                xAxis: 0,
+                yAxis: 0
+            }, {
+                x: 104,
+                y: 96,
+                xAxis: 0,
+                yAxis: 0
+            }, {
+                x: 100,
+                y: 96,
+                xAxis: 0,
+                yAxis: 0
+            }]
+        }],
+        labels: [{
+            text: 'Weakening',
+            backgroundColor: 'transparent',
+            borderWidth: 0,
+            y: 0,
+            style: {
+                fontSize: '16px',
+                fontWeight: 'bold',
+                color: '#ffc600'
+            },
+            point: {
+                x: 104,
+                y: 96,
+                xAxis: 0,
+                yAxis: 0
+            }
+        }]
+    }],
+    title: {
+        text: 'Relative Rotational Graph'
+    },
+    plotOptions: {
+        series: {
+            lineWidth: 2,
+            marker: {
+                enabled: true,
+                radius: 3,
+                symbol: 'circle'
+            }
+        }
+    },
+    tooltip: {
+        pointFormat: 'RS-Ratio: <b>{point.x}</b></br> RS-Momentum: <b>{point.y}</b>'
+    },
+    xAxis: {
+        min: 96,
+        max: 104,
+        plotLines: [{
+            value: 100,
+            width: 2,
+            color: '#000000'
```

Comment:
```suggestion
            color: '#000000',
            zIndex: 1
```
A z-index of 1 makes sure it is not covered by the grid lines